tests/integration/ceph_csi_test.go

```diff
@@ -96,7 +96,7 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
    name: ` + csiSCRBD + `
-provisioner: rbd.csi.ceph.com
+provisioner: rook-ceph.rbd.csi.ceph.com
```

Comment:
question: integration tests will run in there own namespace, an operator will get deployed in smoke-ns-system namespace having `rook-ceph.rbd.csi.ceph.com`  will works?

Reply:
I need to check this. Thanks for pointing it out.

Reply:
I'd be OK with that but my concern is that the way kube works is that if the user doesn't plan ahead and set this variable for their 2nd operator they could end up in a situation where storageclasses that use the "common" rbd.csi.ceph.com would have their messages delivered to the wrong csi/cluster. Putting in the namespace prefix from the get go avoids the chance of these collisions at the cost of being a little more complex. I've seen the confusion that having two provisiioners with the same name can cause so I am personally on the side of keeping the namespace as a default prefix but am willing to be talked out of it.

Reply:
I'm thinking the default should be blank for the following scenario. If someone changes the namespace to something other than rook-ceph, the example storage class won't work and it's not obvious that the prefix needs to match the namespace by default. You can't just search and replace `namespace: rook-ceph` like in other places. So some users could be broken even though they still only have a single namespace. I'd rather make the default always work for the users with a single cluster than try to simplify multiple clusters.